lib/fizzy/parser.cpp

```diff
@@ -285,6 +285,9 @@ Module parse(bytes_view input)
             throw parser_error{"too many memory sections (at most one is allowed)"};
     }
 
+    if (module.startfunc && (*module.startfunc >= module.funcsec.size()))
```

Comment:
I believe this is not correct, because it doesn't take into account the number of imported functions